composer/packages/distribution/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@ballerina/distribution",
+  "version": "2.0.0-Alpha-SNAPSHOT",
+  "description": "",
+  "main": "build/composer.js",
+  "scripts": {
+    "clean": "composer pkg:clean",
+    "lint": "composer pkg:lint",
+    "prebuild": "npm run clean && npm run lint",
+    "test": "echo \"Error: no test specified\"",
+    "test-coverage": "echo \"Error: no test specified\"",
+    "build": "composer pkg:build:webpack",
+    "watch": "composer pkg:watch:webpack"
+  },
+  "author": "ballerina.io",
+  "license": "Apache-2.0",
+  "dependencies": {
+    "@ballerina/theme": "^2.0.0-Alpha-SNAPSHOT",
+    "@ballerina/visualize-syntax-tree": "^2.0.0-Alpha-SNAPSHOT"
+  },
+  "devDependencies": {
+    "@ballerina/composer-cli": "^2.0.0-Alpha-SNAPSHOT"
```

Comment:
Do we use composer-cli?

Reply:
Yes. It is through this that we expose the UI to the client side.